src/main/java/com/wultra/app/enrollmentserver/database/OperationTemplateRepository.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2021 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.app.enrollmentserver.database;
+
+import com.wultra.app.enrollmentserver.database.entity.OperationTemplate;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+/**
+ * Repository for the operation localization templates.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Repository
+public interface OperationTemplateRepository extends CrudRepository<OperationTemplate, Long> {
+
+    Optional<OperationTemplate> findFirstByLanguageAndPlaceholder(String language, String placeholder);
```

Comment:
It is a bit hard to understand what it means to search by `language` and `placeholder`. Is the word `placeholder` the best term to call a localized operation template name? I get it now, but the word placeholder has meanings outside localization and it is not clear that it is a placeholder for an operation name, too. Feel free to ignore.

Reply:
I will probably leave it for now, since calling it "template" would probably cause even more confusion due to different meanings. This is a candidate for the future improvement...